hashintel/hash
# Fix scroll lock issue on entity editor

## 🌟 What is the purpose of this PR?

This PR enables scroll-lock while users are editing value cells on properties table for better UX.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432087/1203568198115136/f) _(internal)_

## 🔍 What does this change?

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor for any entity
2. Edit any value cell
3. Confirm that page scroll is locked while cells are being edited

## 📹 Demo

https://user-images.githubusercontent.com/39553853/210417714-fc1fadb1-0774-4382-a496-2a902ebe98c0.mov
